lib/matplotlib/axes/_axes.py

```diff
@@ -4992,16 +5018,18 @@ def fill_between(self, x, y1, y2=0, where=None, interpolate=False,
             Setting *interpolate* to *True* will calculate the actual
             interscection point and extend the filled region up to this point.
 
-        step : {'pre', 'post', 'mid'}, optional
+        step : [ 'pre' | 'post' | 'mid' ], optional
```

Comment:
The way this was is the numpydoc specified way of listing enumerated values, I think we should be moving to that style and away from the `[ | ]` style (unless I have missed something.

Reply:
Is this consensus? The [documentation guide](https://matplotlib.org/devel/documenting_mpl.html?highlight=documenting) is still using `[ | ]` in its examples.

Reply:
I'd say it should be updated then.